src/core/xfa/bind.js

```diff
@@ -635,6 +635,13 @@ class Binder {
             /* skipConsumed = */ this.emptyMerge
           ).next().value;
           if (!match) {
+            // If there is no match (no data) and `occur.min === 0` then
+            // the container is entirely excluded.
+            // https://www.pdfa.org/norm-refs/XFA-3_3.pdf#G12.1428332
+            if (child.occur.min === 0) {
```

Comment:
I missed that... but `occur` could be null here.
We already get the `min` info here: https://github.com/mozilla/pdf.js/blob/611627f5a1226785a7a952c0fdb6e4e8eba4a2e4/src/core/xfa/bind.js#L560
so you should use this value instead.